media-plugins/kodi-game-libretro-snes9x/kodi-game-libretro-snes9x-1.62.3.50.ebuild

```diff
@@ -24,27 +21,10 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
 RDEPEND="
-	=media-tv/kodi-21*
+	games-emulation/libretro-snes9x
```

Comment:
|| games-emulation/snes9x[libretro]

Reply:
Didn't realise it could be used as a replacement.

What are the drawbacks between using games-emulation/libretro-X vs games-emulation/X[libretro]? 

It appears the latter packages have better maintenance as they are under the games project.

Thinking about the order and if we could stop using games-emulation/libretro-X packages to allow removal.

Reply:
We can't globally remove games-emulation/libretro-X dependencies (at least for some emulators). For snes9x, it's an option.

But for me, we should keep games-emulation/libretro-X as the preferred dependency because it's "just what we need" for Kodi/libretro users. Libretro guys are maintaining a fork and I think this make them more "autonomous" to keep a libretro compatible emulator. Upstream emulator developers don't necessarily focus on libretro compat.

Users that want a more up-to-date or a more featured emulator (or if they use it as a standalone emulator) should explicitly merge it by their own and use games-emulation/X[libretro] if they want to have it as a drop-in replacement of games-emulation/libretro-X package.